File: app/public_html/script/functions/samples/7.c
loop(){
    if(getLowHp()){
        if (getAp() >= 40)
            fireball(getTargetX(), getTargetY());
        else
            attackRanged(getTargetX(), getTargetY());
    }
    else{
        while(getX() != 12.5 || getY() != 12.5)
            teleport(12.5,12.5);
        turn(50);
    }
}

File: app/public_html/script/functions/samples/7.py
def loop():
    if getLowHp():
        if getAp() >= 40:
            fireball(getTargetX(), getTargetY())
        else:
            attackRanged(getTargetX(), getTargetY())
     
    else:
        while getX() != 12.5 or getY() != 12.5:
            teleport(12.5,12.5)
        turn(50)
     
 